types/chrome/chrome-cast/index.d.ts

```diff
@@ -843,10 +843,10 @@ declare namespace chrome.cast.media {
         streamType: chrome.cast.media.StreamType;
         contentType: string;
         metadata: any;
-        duration: number;
-        tracks: chrome.cast.media.Track[];
-        textTrackStyle: chrome.cast.media.TextTrackStyle;
-        customData: Object;
+        duration?: number;
+        tracks?: chrome.cast.media.Track[];
```

Comment:
I don't think "nullable" in the docs means `| null`, I interpret it as "a value may be missing" (nullish). I haven't seen most of these fields having either a `null` or `undefined` value, I don't even use some of them. But `tracks`, I know for a fact is sometimes `undefined`. Never `null`, AFAIK. It is possible that some of them use `null` instead of `undefined`, I don't know. But signaling in the type definitions that they might not have a value, makes the current definitions less wrong and avoids runtime errors for anyone checking for "truthyness" (`if (value)`) or nullishness (`value ?? fallback`).

Do you know of a reliable way to check which ones use `null` or `undefined`?

Reply:
Another alternative, if there's no reliable way to know which may be `undefined` or `null`, is to assume all of them can be both. That would be the safest alternative.

```ts
tracks?: chrome.cast.media.Track[] | null;
```